code/modules/shuttle/emergency.dm

```diff
@@ -86,13 +86,13 @@
 
 	if((old_len != authorized.len) && !ENGINES_STARTED)
 		var/alert = (authorized.len > old_len)
+		var/repeal = (authorized.len < old_len)
 		var/remaining = auth_need - authorized.len
 		if(authorized.len && remaining)
 			minor_announce("[remaining] authorizations \
```

Comment:
Do you mean add both vars to the same line like var/alert = (authorized.len > old_len), var/repeal = (authorized.len < old_len) ?